gremlin-go/README.md

```diff
@@ -91,65 +121,190 @@ Note: The exact import name as well as the module prefix for `NewDriverRemoteCon
 data-flow language that enables users to succinctly express complex traversals on (or queries of) their application's
 property graph.
 
-Gremlin-Go implements Gremlin within the Go language and can be used on any Go runtime greater than v1.17. Go's syntax 
-has the same constructs as Java including "dot notation" for function chaining (a.b.c), round bracket function arguments
-(a(b,c)), and support for global namespaces (a(b()) vs a(__.b())). One important distinction with Go and Java is that 
-the functions are capitalized, as is required to export functions is Go. As such, anyone familiar with Gremlin-Java 
-will immediately be able to work with Gremlin-Go.
+Gremlin-Go implements Gremlin within the Go language and can be used on any Go runtime greater than v1.17. One important distinction with Go and Java is that 
+the functions are capitalized, as is required to export functions is Go. 
```

Comment:
nitpick: it looks like the existing readme's text was wrapped at ~120 characters. Might want to do that here as well?